More improvements for S3, files scan and BG jobs (#768)
owncloud/docs-server: modules/admin_manual/pages/configuration/files/external_storage/configuration.adoc

* Rescanning S3 for manually added files is not possible
+

> NOTE: The command xref:configuration/server/occ_command.adoc#the-filesscan-command[occ files:scan] respectively any xref:configuration/server/occ_command.adoc#managing-background-jobs[background job] containing _files scan_ is only available for POSIX but not for S3 and compatible filesystems. See the  for more information.

Reviewer: ```suggestion
NOTE: The command xref:configuration/server/occ_command.adoc#the-filesscan-command[occ files:scan] or a xref:configuration/server/occ_command.adoc#managing-background-jobs[background job] containing _files scan_ is only available for POSIX but not for S3 and compatible filesystems.
```
